### PR TITLE
Merge LD material

### PR DESCRIPTION
**Pull request by deflaux**
This was previously reviewed via https://github.com/googlegenomics/readthedocs/pull/117 and https://github.com/googlegenomics/readthedocs/pull/120

### CODE REVIEW

File: docs/source/github_index.rst [hunk 1]
```diff
@@ -77,11 +74,11 @@ Data Analysis Application Samples
 `denovo-variant-caller-java <https://github.com/googlegenomics/denovo-variant-caller-java>`_
   A de novo variant caller which uses information from a mother, father and child trio with a bayesian inference method.
```
Comment (pgrosu) [inline docs/source/github_index.rst]: s/bayesian/Bayesian

File: docs/source/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+.. toctree::
+   :maxdepth: 2
+
+.. contents::
+
+This pipeline calculates linkage disequilibrium between pairs of variants in a Variant Set. It takes as input a VariantSet for which the linkage disequilibrium values will be calculated and calculates the D' and allelic correlation measures of linkage disequilibrium, defined in
```
Comment (pgrosu) [inline docs/source/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium.rst]: s/Variant Set/VariantSet

Comment (jbingham) [inline docs/source/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium.rst]: add link on VariantSet to https://cloud.google.com/genomics/reference/rest/v1/variantsets 

in a Global Alliance VariantSet (which you can create from a VCF file https://cloud.google.com/genomics/v1/load-variants).

File: docs/source/use_cases/linkage_disequilibrium/transform_ld_results.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+.. toctree::
+   :maxdepth: 2
+
+.. contents::
+
+These pipelines take linkage disequilibrium (LD) results generated by the :doc:`/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium` pipeline and transforms them into `Cloud BigQuery <https://cloud.google.com/bigquery/>`_ and `Cloud BigTable <https://cloud.google.com/bigtable/docs/>`_ datasets that can be efficiently analyzed. Each pipeline takes as input a set of LD results and exports, transforms, and loads the results into the appropriate target data store.
```
Comment (jbingham) [inline docs/source/use_cases/linkage_disequilibrium/transform_ld_results.rst]: s/transforms/transform/ (it's "these pipelines" that transform)

**Comment by jbingham** [comment]
Just a couple tiny comments.

**Pull request merged**